library/lib/serve.js

```diff
@@ -84,7 +77,7 @@ async function resolveFile(req, res, next) {
     const { path } = req
     /** @type {Array<[string, (file:any) => any]>} */
     const combinations = [
-      [indexWithRouting, file => serveIndexWithRouting(req, res, file)],
+      [indexWithRouting, file => serveIndexWithRouting(file, { req, res, next })],
```

Comment:
Ik zou hier geen object aanmaken
```suggestion
      [indexWithRouting, file => serveIndexWithRouting(file, req, res, next)],
```